lametro/search_indexes.py

```diff
@@ -29,7 +29,8 @@ def prepare_controlling_body(self, obj):
         return None
 
     def prepare_sponsorships(self, obj):
-        return [action.organization for action in obj.actions.all()]
+        orgs_list = [action.organization for action in obj.actions_and_agendas]
```

Comment:
Getting `AttributeError: 'dict' object has no attribute 'organization'` when I try to update the index. Isn't `actions_and_agendas` a list of dictionaries?

Reply:
woops, yup, it is! notation error on my part.